commit 02a4b1a66d634aa1506e29372cbea24e9a61523f (refs/changes/90/790/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-10-27T01:38:29+00:00 (2 years, 11 months ago)
    
    PUBAPI-1343 cueball update made startup a bit slower
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index 1d8fc06..a3c871e 100644
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
         "nopt": "2.0.0",
         "restify": "git+https://github.com/mcavage/node-restify.git#0d7b4ba",
         "bunyan": "0.22.1",
-        "cueball": "1.1.2",
+        "cueball": "1.1.4",
         "kang": "1.1.0",
         "sdc-clients": "9.4.0",
         "ufds": "1.1.3",
